lib/cli/daemoncommand.cpp

```diff
@@ -203,6 +234,16 @@ static Atomic<bool> l_AllowedToWork (false);
 static inline
 int RunWorker(const std::vector<std::string>& configs, bool closeConsoleLog = false, const String& stderrFile = String())
 {
+
+#ifdef I2_DEBUG
+	double delay = GetDebugWorkerDelayCached();
```

Comment:
IMAO if you're using it just here, it's not worth caching. Oh and... you actually aren't caching across reloads.

Reply:
Imho a minor problem with the real problem it is solving for debugging.

Reply:
IMAO a valid argument for not fixing present problems or no-ops, but not for introducing already known problems or no-ops. AFAIK no call will run [here](https://github.com/Icinga/icinga2/pull/7793/files#diff-e8b23cce1118fe8cabeb821498fef0cbR205), so we shouldn't introduce that no-op mechanism.